src/MessagingThreadHistory/MessagingThreadHistory.tsx

```diff
@@ -27,30 +28,69 @@ interface Props {
   messages: MessageData[];
 }
 
-export const MessagingThreadHistory: React.FC<Props> = ({
-  className,
-  threadID,
-  messages,
-}: Props) => {
-  const lastMessageRef = useRef<HTMLDivElement>(null);
-  const messagesWithDividers = _interleaveMessagesWithDividers(messages, lastMessageRef);
-
-  // We want to scroll to the bottom if there is a new message (received in real-time, or
-  //  user just sent one themselves) or if the user switches to another thread. Cannot rely
-  //  on messages themselves (new reference each render) nor on message count alone (different
-  //  threads can have the same message count). Instead, need the threadID AND message count.
-  useEffect(() => {
-    if (messages.length > 0) {
-      lastMessageRef.current.scrollIntoView();
-    }
-  }, [messages.length, threadID]);
+const SCROLL_BUFFER = 200;
 
+// Always returns true with current mobile strategy of scrolling the entire thread history
```

Comment:
This isn't ideal, but with how different our mobile and desktop layouts are, we are going to need to figure out how we want to approach things like scrolling in a uniform way.